.gitignore

```diff
@@ -16,5 +16,4 @@ yarn.lock
 *.DS_Store
 
 # Build directories (Will be preserved by npm)
-dist
```

Comment:
The `npm publish` operation only looks at `.gitignore` in the case where an `.npmignore` file does not exist ([npm docs](https://docs.npmjs.com/misc/developers#keeping-files-out-of-your-package)). Build files are also regenerated at publish time with the package `prepublish`, so they are best left out of the git repo.